Source/evac.f90

```diff
@@ -1472,6 +1474,10 @@ SMOKE_KS_SPEED_FUNCTION = 1 ! Linear function c(Ks) = (1 + beta*Ks/alpha)
       ! c(Ks) = ( 1 + (BETA*Ks)/ALPHA )  [0,1] interval, c(Ks=0)=1, c(Ks=inf)=0
       SMOKE_SPEED_ALPHA = 0.706_EB   ! Lund 2003, report 3126 (Frantzich & Nilsson)
       SMOKE_SPEED_BETA  = -0.057_EB  ! Lund 2003, report 3126 (Frantzich & Nilsson)
+	  
+	  ! Tune mobility of agent in pre-evacuation time period
```

Comment:
What do you mean with "_tune mobility_"?

Reply:
> Tune mobility of agent in pre-evacuation time period

In brief the idea is enabling agent to move around in pre-evacuaiton period.  In current version, agents stand still if T<TPRE, which means agents cannot move in pre-evacuation period.  If you are interested, I will provide you with a test case and you will get the problem.

Reply:
see above